spec/models/contact_spec.rb

```diff
@@ -147,73 +143,70 @@
       end
 
       it 'returns no duplicate contacts' do
-        [@alice, @eve].each {|c| @bob.add_contact_to_aspect(@bob.contact_for(c.person), @bob.aspects.last)}
+        [alice, eve].each {|c| bob.add_contact_to_aspect(bob.contact_for(c.person), bob.aspects.last)}
         contact_ids = @contact.contacts.map{|p| p.id}
         expect(contact_ids.uniq).to eq(contact_ids)
       end
     end
 
     context 'on a contact for a remote user' do
-      before do
-        @contact = @bob.contact_for @people1.first
-      end
+      let(:contact) { bob.contact_for @people1.first }
+
       it 'returns an empty array' do
-        expect(@contact.contacts).to eq([])
+        expect(contact.contacts).to eq([])
       end
     end
   end
 
   context 'requesting' do
-    before do
-      @contact = Contact.new
-      @user = FactoryGirl.build(:user)
-      @person = FactoryGirl.build(:person)
+    let(:contact) { Contact.new }
+    let(:user)    { build(:user) }
+    let(:person)  { build(:person) }
 
-      @contact.user = @user
-      @contact.person = @person
+    before do
+      contact.user = user
+      contact.person = person
```

Comment:
The real benefit of let, lazy evaluation, feels a bit lost here. Doesn't doing `Contact.new user: user, person: person` work?